src/cli/modules.c

```diff
@@ -67,11 +67,19 @@ typedef struct
   ClassRegistry classes[MAX_CLASSES_PER_MODULE];
 } ModuleRegistry;
 
+// Visual C++ doesn't support initializing arrays with {} but we need to use it
+// for compilers that actually follow the standard.
+#ifndef _MSC_VER
+  #define EMPTY {}
+#else
+  #define EMPTY {0}
+#endif
```

Comment:
Aw, man, VC++ is always a pain. I tried a different approach to this code that I think avoids this problem. Can you merge in 69a5c1d017ca185e6466eb848cc76cff55aa60b2 and see if that helps?